book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(person, date)
+    Rental.new(date, self, person)
```

Comment:
- [ ] Your arrangement of parameters here is wrong. It should uniform with the rental class constructor:

  ```
  Rental.new(date, person, self)
  ```